src/transaction/log_2pc.c

```diff
@@ -65,15 +66,23 @@
 /* Variables */
 struct log_2pc_global_data
 {
-  int (*get_participants) (int *particp_id_length, void **block_particps_ids);
+  int (*get_participants) (THREAD_ENTRY * thread_p, int *particp_id_length, void **block_particps_ids);
   int (*lookup_participant) (void *particp_id, int num_particps, void *block_particps_ids);
-  char *(*sprintf_participant) (void *particp_id);
   void (*dump_participants) (FILE * fp, int block_length, void *block_particps_id);
-  int (*send_prepare) (int gtrid, int num_particps, void *block_particps_ids);
-    bool (*send_commit) (int gtrid, int num_particps, int *particp_indices, void *block_particps_ids);
-    bool (*send_abort) (int gtrid, int num_particps, int *particp_indices, void *block_particps_ids, int collect);
+    bool (*send_prepare) (THREAD_ENTRY * thread_p, int gtrid, int num_particps, void *block_particps_ids);
+  void (*send_commit) (THREAD_ENTRY * thread_p, int gtrid, int num_particps, bool is_commit, void *block_particps_ids);
+  void (*send_abort) (THREAD_ENTRY * thread_p, int gtrid, int num_particps, bool is_commit, void *block_particps_ids);
```

Comment:
abort의 경우 is_abort 인자가 더 직관성 있어 보입니다.
```suggestion
  void (*send_abort) (THREAD_ENTRY * thread_p, int gtrid, int num_particps, bool is_abort, void *block_particps_ids);
```